test/Dapper.AOT.Test/SamplesTests.cs

```diff
@@ -47,8 +47,8 @@ public void Run(string path)
             var actual = sb.ToString();
             try // automatically overwrite test output, for git tracking
             {
-                if (GetOriginCodeLocation() is string originFile
-                    && Path.GetDirectoryName(originFile) is string originFolder)
+                if (GetOriginCodeLocation() is { } originFile
+                    && Path.GetDirectoryName(originFile) is { } originFolder)
```

Comment:
nit: subjective, but I think I prefer the `is string` syntax (or `is not null` if we aren't capturing the value) when we're not using the `{...}` to test members - for clarity reasons